Fitness/Views/ProRoutineView.swift

```diff
@@ -0,0 +1,117 @@
+    var addButton: UIButton!
+    var stepsTextView: UITextView!
+    
+    init(routine: ProRoutine) {
+        super.init(frame: .zero)
+        
+        self.backgroundColor = .white
+        
+        titleLabel = UILabel()
+        titleLabel.textColor = .black
+        titleLabel.font = ._16MontserratMedium
+        titleLabel.textAlignment = .left
+        titleLabel.text = routine.title
+        titleLabel.sizeToFit()
+        self.addSubview(titleLabel)
+        
+        routineTypeImage = UIImageView()
+        routineTypeImage.contentMode = .scaleAspectFit
+        routineTypeImage.clipsToBounds = true
+        self.addSubview(routineTypeImage)
+        
+        routineTypeLabel = UILabel()
+        routineTypeLabel.textColor = UIColor.colorFromCode(0x555555)
+        routineTypeLabel.font = ._12MontserratRegular
+        routineTypeLabel.textAlignment = .center
+        routineTypeLabel.sizeToFit()
+        self.addSubview(routineTypeLabel)
+        
+        switch routine.routineType {
+        case .cardio:
+            routineTypeImage.image = #imageLiteral(resourceName: "small cardio icon.png")
+            routineTypeLabel.text = "Cardio"
+        case .mindfullness:
+            routineTypeImage.image = #imageLiteral(resourceName: "smaller mindfulness icon.png")
+            routineTypeLabel.text = "Mindfulness"
+        case .strength:
+            routineTypeImage.image = #imageLiteral(resourceName: "small strength icon.png")
+            routineTypeLabel.text = "Strength"
+        case .other:
+            routineTypeImage.image = #imageLiteral(resourceName: "smaller mindfulness icon.png")
+            routineTypeLabel.text = "Other"
+        }
+        
+        addButton = UIButton()
+        addButton.setBackgroundImage(#imageLiteral(resourceName: "add habit.png"), for: .normal)
+//        self.addSubview(addButton)
+        //Commenting out the add button until more functionality to save and display favourited routines is implemented
+        
+        stepsTextView = UITextView()
+        stepsTextView.textColor = UIColor.colorFromCode(0x222222)
+        stepsTextView.font = ._14MontserratRegular
+        stepsTextView.isScrollEnabled = false
+        stepsTextView.textAlignment = .left
+        stepsTextView.isSelectable = false
+        stepsTextView.text = routine.text
+        stepsTextView.sizeToFit()
+        self.addSubview(stepsTextView)
+        
+    }
+    
+    override func updateConstraints() {
+        titleLabel.snp.makeConstraints{make in
+            make.leading.equalToSuperview().offset(16)
+            make.top.equalToSuperview().offset(16)
+            make.trailing.equalToSuperview().offset(-16)
+        }
+        
+        routineTypeImage.snp.makeConstraints{make in
+            make.height.width.equalTo(15)
+            make.top.equalTo(titleLabel.snp.bottom).offset(3)
+            make.leading.equalTo(titleLabel)
+        }
+        
+        routineTypeLabel.snp.makeConstraints{make in
```

Comment:
indentation here is wrong again, space after `{`

Reply:
I'll change them